code/datums/components/crafting/recipes/carnival_silkcrafting.dm

```diff
@@ -411,7 +457,7 @@
 	category = CAT_SILK
 
 /datum/crafting_recipe/ncorpcommander
-	name = "R�stung der auserw�hlten Frau Gottes"
+	name = "R�stung der auserw�hlten Frau Gottes"
```

Comment:
Is this a intended change or is it the code messing up.

Reply:
just the code being a bit stinky, still loads in as normal ingame from what ive seen

Reply:
I think it is the code messing up.